test/unit/specs/components/Identity/getIdentity/createGetIdentity.spec.js

```diff
@@ -0,0 +1,51 @@
+import createGetIdentity from "../../../../../../src/components/Identity/getIdentity/createGetIdentity";
+
+describe("Identity::createGetIdentity", () => {
+  let sendEdgeNetworkRequest;
+  let createIdentityPayload;
+  const samplePayload = {
+    myPayload: { methodOne: () => {}, methodTwo: () => {} }
+  };
+  beforeEach(() => {
+    sendEdgeNetworkRequest = jasmine.createSpy("sendEdgeNetworkRequest");
+    createIdentityPayload = jasmine
+      .createSpy("createIdentityPayload")
+      .and.returnValue(samplePayload);
+  });
+  it("should return a function which calls sendEdgeNetworkRequest", () => {
+    const getIdentity = createGetIdentity({
+      sendEdgeNetworkRequest,
+      createIdentityPayload
+    });
+    expect(typeof getIdentity).toBe("function");
+    getIdentity();
+    expect(sendEdgeNetworkRequest).toHaveBeenCalledWith({
+      payload: samplePayload,
+      action: "identity/acquire"
+    });
+  });
+
+  it("Each getIdentity call should create a new payload object", () => {
```

Comment:
Try to be consistent with capitalization between tests. I'm mostly seen the first character in the description lower-cased, but we seem to be inconsistent across the team.